class UnrecoverableError(Exception):
    pass


class RevokeChain(Exception):
    pass
